hugo/data/teams.yml

```diff
@@ -126,6 +130,10 @@ sections:
           name: Jan Hůla
           position: Machine Learning Intern
           github: https://github.com/Jan21
+        - picture: placeholder/male.png
+          name: Tristan Kalos
+          position: Machine Learning Intern
+          github: https://github.com/glimow
```

Comment:
@vmarkovtsev is that one https://github.com/glimow our Tristan Kalos ?

Reply:
IDK, he did not share his GitHub with me. Looks very truthful.

Reply:
Last question; when is he joining us @vmarkovtsev ?

Reply:
@esther knows, I don't...